### PR TITLE
fixed typos and indentation in _spam.html.erb

### LINKED ISSUE #10984: Fix typos and indentation on _spam.html.erb
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all), and if fixed, will help resolve part of the issue at #10952. We've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new** and looking through our general [help wanted](https://github.com/publiclab/plots2/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Claim the issue by commenting. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!
<details>
  <summary>💡 <i>Learn how to claim</i> 🙋</summary>
  <h3>Claiming an issue</h3>
  <p>Unless the issue is marked as reserved for someone, you can just say "I'd like to try this!" and then you've claimed it - no need to wait for someone to assign it to you. Just be sure you link your pull request (PR) to this issue so we can see where your solution is.</p>
  <p>And open one early if possible - even before you've completed it with additional commits - and others can help you figure out any issues you may face.</p>
</details>

- [ ] 📝 **Update** the file [_spam.html.erb](https://github.com/publiclab/plots2/blob/main/app/views/spam2/_spam.html.erb) in the `plots2` repository (press the little pen Icon) and edit the line as shown below.

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -24,18 +24,18 @@
     </a>
     <div class="dropdown-menu dropdown-menu-right">
       <h6 class="dropdown-header">Menu | <%= current_user.role%> </h6>
-        <a class="dropdown-item <% if params[:action] == "_spam_insights" %> active <% end %>" href="/spam2/insights"><i class="drop fa fa-line-chart text-primary"></i> Insights </a>
+      <a class="dropdown-item <% if params[:action] == "_spam_insights" %> active <% end %>" href="/spam2/insights"><i class="drop fa fa-line-chart text-primary"></i> Insights </a>
       <div class="dropdown-divider"></div>  
-        <h6 class="dropdown-header">Spam Moderation</h6>
-        <a class="drop_click dropdown-item <% unless  params[:action] =="_spam_comments" ||params[:action] =="_spam_queue" || params[:action] =="_spam_insights"|| params[:action] =="_spam_revisions" || params[:action] =="_spam_flags" || params[:type] == "wiki" || params[:action] == "_spam_users"%> active <% end %>" href="/spam2/filter/all/30"><i class="drop fa fa-address-book text-primary"></i> Moderation</a>
-        <a class="wiki_drop_click dropdown-item <% if params[:type] == "wiki" %> active <% end %>" id="wiki_drop" href="/spam2/filter/wiki/30" ><i class="drop fa fa-book text-primary"></i> Wiki</a>
-        <a class="drop_click dropdown-item <% if params[:action] == "_spam_queue" %> active <% end %>" href="/spam2/queue/filter/everything" ><i class="drop fa fa-clock text-primary"></i> Queue</a>
-        <a class="drop_click dropdown-item <% if params[:action] == "_spam_revisions" %> active <% end %>" href="/spam2/revisions"><i class="drop fa fa-list text-primary"></i> Revision</a>
-        <a class="drop_click dropdown-item <% if params[:action] == "_spam_comments" %> active <% end %>" href="/spam2/comments/filter/all/30" ><i class="drop fa fa-comment text-primary"></i> Comments</a>
-        <a class="drop_click dropdown-item <% if params[:action] == "_spam_flags" %> active <% end %>"  href="/spam2/flags/filter/all/30" ><i class="drop fa fa-flag text-primary"></i> Flag </a>
-        <a class="drop_click dropdown-item  <% if params[:action] == "_spam_users" %> active <% end %>" href="/spam2/users/filter/all/30"><i class="drop fa fa-user text-primary"></i> User moderation</a>
-      </div>
+      <h6 class="dropdown-header">Spam Moderation</h6>
+      <a class="drop_click dropdown-item <% unless  params[:action] =="_spam_comments" ||params[:action] =="_spam_queue" || params[:action] =="_spam_insights"|| params[:action] =="_spam_revisions" || params[:action] =="_spam_flags" || params[:type] == "wiki" || params[:action] == "_spam_users"%> active <% end %>" href="/spam2/filter/all/30"><i class="drop fa fa-address-book text-primary"></i> Moderation</a>
+      <a class="wiki_drop_click dropdown-item <% if params[:type] == "wiki" %> active <% end %>" id="wiki_drop" href="/spam2/filter/wiki/30" ><i class="drop fa fa-book text-primary"></i> Wiki</a>
+      <a class="drop_click dropdown-item <% if params[:action] == "_spam_queue" %> active <% end %>" href="/spam2/queue/filter/everything" ><i class="drop fa fa-clock text-primary"></i> Queue</a>
+      <a class="drop_click dropdown-item <% if params[:action] == "_spam_revisions" %> active <% end %>" href="/spam2/revisions"><i class="drop fa fa-list text-primary"></i> Revision</a>
+      <a class="drop_click dropdown-item <% if params[:action] == "_spam_comments" %> active <% end %>" href="/spam2/comments/filter/all/30" ><i class="drop fa fa-comment text-primary"></i> Comments</a>
+      <a class="drop_click dropdown-item <% if params[:action] == "_spam_flags" %> active <% end %>"  href="/spam2/flags/filter/all/30" ><i class="drop fa fa-flag text-primary"></i> Flag </a>
+      <a class="drop_click dropdown-item  <% if params[:action] == "_spam_users" %> active <% end %>" href="/spam2/users/filter/all/30"><i class="drop fa fa-user text-primary"></i> User moderation</a>
     </div>
+  </div>
 </div>
 <script>
   $(document).on('click', '.drop_click', function(){
@@ -62,7 +62,7 @@
 </div>
 
 <div class="collapse" id="info-div">
-  <p class="text-secondary font-italic"><i class="fa fa-certificate text-info" aria-hidden="true"></i> Moderators and admins have the ability to "spam" posts if they include inappropriate content, blatant advertising, or are otherwise problematic. If you're not sure, email at <a href="mailto:organizers@<%= request.host %>" class="badge badge-info badge-pill text-white">moderators@<%= request.host %></a> If we all work together, we can keep spam to a minimum. Thanks for helping out!</p>
+  <p class="text-secondary font-italic"><i class="fa fa-certificate text-info" aria-hidden="true"></i> Moderators and admins have the ability to "spam" posts if they include inappropriate content, blatant advertising, or are otherwise problematic. If you're not sure, email at <a href="mailto:organizers@<%= request.host %>" class="badge badge-info badge-pill text-white">moderators@<%= request.host %></a>. If we all work together, we can keep spam to a minimum. Thanks for helping out!</p>
 </div>
 
 <nav class="navbar navbar-expand " id="bulk-nav">
@@ -70,50 +70,50 @@
     <ul class=" navbar-nav">
       <% if params[:action] != "_spam_users" && params[:action] != "_spam_revisions" && params[:action] != "_spam_insights"%>
         <li class="nav-item pr-4">
-          <a class="nav-link btn btn-light border disabled text-dark d-inline-flex border-curve" id="batch-spam" ><i class=" pt-1 pr-2 fa fa-ban text-danger" data-toggle="tooltip" data-placement="top" title="Mark selected items as spam and ban users"></i>  Spam</a>
+          <a class="nav-link btn btn-light border disabled text-dark d-inline-flex border-curve" id="batch-spam" ><i class=" pt-1 pr-2 fa fa-ban text-danger" data-toggle="tooltip" data-placement="top" title="Mark selected items as Spam and ban users"></i>  Spam</a>
         </li>
         <li class="nav-item pr-4">
-          <a class="nav-link btn btn-light border disabled text-dark d-inline-flex border-curve"  id="batch-publish" ><i class=" pt-1 pr-2 fa fa-check-circle text-success" data-toggle="tooltip" data-placement="top" title="Mark selected items  as Publish and unaban users"></i>  Publish</a>
+          <a class="nav-link btn btn-light border disabled text-dark d-inline-flex border-curve"  id="batch-publish" ><i class=" pt-1 pr-2 fa fa-check-circle text-success" data-toggle="tooltip" data-placement="top" title="Mark selected items as Publish and unban users"></i>  Publish</a>
         </li>
         <li class="nav-item pr-4">
-          <a class="nav-link btn btn-light border disabled text-dark d-inline-flex border-curve" id="delete-batch"><i class=" pt-1 pr-2 fa fa-trash text-danger" data-toggle="tooltip" data-placement="top" title="Delete Selected items"></i> Delete</a>
+          <a class="nav-link btn btn-light border disabled text-dark d-inline-flex border-curve" id="delete-batch"><i class=" pt-1 pr-2 fa fa-trash text-danger" data-toggle="tooltip" data-placement="top" title="Delete selected items"></i> Delete</a>
         </li>
-        <% end %>
+      <% end %>
       
-        <% if params[:action] != "_spam_insights" %>
+      <% if params[:action] != "_spam_insights" %>
         <li class="nav-item pr-4">
-          <a class="nav-link btn btn-light border text-dark d-inline-flex border-curve"  id="all" ><i class=" pt-1 pr-2 fa fa-check text-primary" data-toggle="tooltip" data-placement="top" title="Select all in this page"></i> Select</a>
+          <a class="nav-link btn btn-light border text-dark d-inline-flex border-curve"  id="all" ><i class=" pt-1 pr-2 fa fa-check text-primary" data-toggle="tooltip" data-placement="top" title="Select all on this page"></i> Select</a>
         </li>
-        <% end %>
+      <% end %>
       
-        <% if params[:action] != "_spam_revisions" && params[:action] != "_spam_insights" &&  params[:action] != "_spam_comments" %>
+      <% if params[:action] != "_spam_revisions" && params[:action] != "_spam_insights" &&  params[:action] != "_spam_comments" %>
         <li class="nav-item pr-4">
-          <a class="nav-link btn btn-light border disabled text-dark d-inline-flex border-curve" id="batch-ban" ><i class=" pt-1 pr-2 fa fa-flag text-warning" data-toggle="tooltip" data-placement="top" title="Ban Users"></i> Ban</a>
+          <a class="nav-link btn btn-light border disabled text-dark d-inline-flex border-curve" id="batch-ban" ><i class=" pt-1 pr-2 fa fa-flag text-warning" data-toggle="tooltip" data-placement="top" title="Ban users"></i> Ban</a>
         </li>
         <li class="nav-item pr-4">
-          <a class="nav-link btn btn-light border disabled text-dark d-inline-flex border-curve" id="batch-unban" ><i class=" pt-1 pr-2 fa fa-flag text-success" data-toggle="tooltip" data-placement="top" title="Unban Users"></i> Unban</a>
+          <a class="nav-link btn btn-light border disabled text-dark d-inline-flex border-curve" id="batch-unban" ><i class=" pt-1 pr-2 fa fa-flag text-success" data-toggle="tooltip" data-placement="top" title="Unban users"></i> Unban</a>
         </li>
         <li class="nav-item pr-4">
           <a href="https://publiclab.org/notes/keshav_sethi0004/08-18-2020/spam-management-dashboard-documentation" target="_blank" class="nav-link btn btn-light border d-inline-flex bo
rder-curve text-dark" id="docs-btn" ><i class=" pt-1 pr-2 fa fa-info-circle text-info" data-toggle="tooltip"  data-placement="top" title="Documentation of Spam2"></i> Docs</a>
         </li>
-        <% end %>
+      <% end %>
     </ul>
   </div>
 </nav>
 
 <% if params[:action] != "_spam_revisions" && params[:action] != "_spam_insights" && params[:action] != "_spam_queue"%>
-<div class="w-20 d-flex ml-1">
-  <a class="btn nav-link text-secondary font-weight-bold">Nodes per page </a>
-  <select name="Visible" id="pageselect" class="overflow-hiden mt-1 custom-select custom-select-sm form-control form-control-sm">
-    <option value="30">30</option>
-    <option value="50">50</option>
-    <option value="100">100</option>
-    <option value="200">200</option>
-    <option value="400">400</option>
-    <option value="600">600</option>
-    <option value="1000">1000</option>
-  </select> 
-</div>
+  <div class="w-20 d-flex ml-1">
+    <a class="btn nav-link text-secondary font-weight-bold">Nodes per page </a>
+    <select name="Visible" id="pageselect" class="overflow-hidden mt-1 custom-select custom-select-sm form-control form-control-sm">
+      <option value="30">30</option>
+      <option value="50">50</option>
+      <option value="100">100</option>
+      <option value="200">200</option>
+      <option value="400">400</option>
+      <option value="600">600</option>
+      <option value="1000">1000</option>
+    </select> 
+  </div>
 <% end %>
 
 <%= render partial: 'spam2/nodes' if @nodes %>
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://app.egghead.io/playlists/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there.

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance!

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it.

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below, and [reach out in our chatroom](https://publiclab.org/chat)!

### PR DESCRIPTION
**Pull request by imranburki**
Fixes #10984 

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [ ] PR is descriptively titled 📑 and links the original issue above 🔗
* [ ] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [ ] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

### CODE REVIEW

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: app/views/spam2/_spam.html.erb [hunk 1]
```diff
@@ -60,60 +62,61 @@
...
+  </p>
 </div>
 
 <nav class="navbar navbar-expand " id="bulk-nav">
   <div class="navbar-expand navbar-nav-scroll d-inline">
     <ul class=" navbar-nav">
       <% if params[:action] != "_spam_users" && params[:action] != "_spam_revisions" && params[:action] != "_spam_insights"%>
         <li class="nav-item pr-4">
-          <a class="nav-link btn btn-light border disabled text-dark d-inline-flex border-curve" id="batch-spam" ><i class=" pt-1 pr-2 fa fa-ban text-danger" data-toggle="tooltip" data-placement="top" title="Mark selected items as spam and ban users"></i>  Spam</a>
+          <a class="nav-link btn btn-light border disabled text-dark d-inline-flex border-curve" id="batch-spam" ><i class=" pt-1 pr-2 fa fa-ban text-danger" data-toggle="tooltip" data-placement="top" title="Mark selected items as Spam and ban users"></i>Spam</a>
```
Comment (TildaDares) [inline app/views/spam2/_spam.html.erb:77]:
```suggestion
          <a class="nav-link btn btn-light border disabled text-dark d-inline-flex border-curve" id="batch-spam" ><i class=" pt-1 pr-2 fa fa-ban text-danger" data-toggle="tooltip" data-placement="top" title="Mark selected items as spam and ban users"></i>Spam</a>
```

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: app/views/spam2/_spam.html.erb [hunk 2]
```diff
@@ -60,60 +62,61 @@
...
+        <select name="Visible" id="pageselect" class="overflow-hidden mt-1 custom-select custom-select-sm form-control form-control-sm">
+          <option value="30">30</option>
+          <option value="50">50</option>
+          <option value="100">100</option>
+          <option value="200">200</option>
+          <option value="400">400</option>
+          <option value="600">600</option>
+          <option value="1000">1000</option>
+      </select> 
+    </div>
```
Comment (TildaDares) [inline app/views/spam2/_spam.html.erb:118]:
```suggestion
```

**Review by TildaDares — CHANGES_REQUESTED** [review]

File: app/views/spam2/_spam.html.erb [hunk 3]
```diff
@@ -60,60 +66,63 @@
...
+        <select name="Visible" id="pageselect" class="overflow-hidden mt-1 custom-select custom-select-sm form-control form-control-sm">
+          <option value="30">30</option>
+          <option value="50">50</option>
+          <option value="100">100</option>
+          <option value="200">200</option>
+          <option value="400">400</option>
+          <option value="600">600</option>
+          <option value="1000">1000</option>
+      </select> 
+
```
Comment (TildaDares) [inline app/views/spam2/_spam.html.erb:124]:
```suggestion
```

**Review by TildaDares — CHANGES_REQUESTED** [review]

**Comment by TildaDares** [comment]
@imranburki I think this is the only thing left to do.

> Also, can you remove `package.json` and `yarn.lock` from this PR? Thanks!

**Comment by imranburki** [comment]
@TildaDares I deleted/removed yarn.lock and package.json . It show conflict error. please help me what should i do?

**Comment by TildaDares** [comment]
@imranburki You're not supposed to delete the files. All you have to do is copy the contents of `yarn.lock` and `package.json` from this repo and paste into your local files. After that, do a push.

**Comment by TildaDares** [comment]
@imranburki I tried removing `yarn.lock` from my end but it didn't work. What you have to do is copy the contents of `yarn.lock` from https://github.com/publiclab/plots2/edit/main/yarn.lock and paste it into your local version of `yarn.lock`. After that do a push.

**Comment by TildaDares** [comment]
@imranburki Is it possible to close this PR and start afresh?

**Comment by imranburki** [comment]
@TildaDares yes, please

**Comment by imranburki** [comment]
closed pull request

**Pull request closed without merging**